lib/views/log-message.js

```diff
@@ -5,42 +5,54 @@ import etch from 'etch'
 import path from 'path'
 import { Range } from 'atom'
 
+function fileRef (filePath, range, bracket) {
```

Comment:
Can we give this function a more descriptive name? Something along the lines of `fileReferenceElement` or similar. Open to other suggestions, I just want to avoid unnecessary shortening of variable, parameter, and function names.